### ISSUE TITLE
Simulator timeout for Scan

### ISSUE THREAD

**Issue by jcampbell05** [fastlane/fastlane#3921]
### New Issue Checklist

- [x ] Updated fastlane to the latest version
- [x ] I have read the [Contribution Guidelines](https://github.com/fastlane/fastlane/blob/master/CONTRIBUTING.md)

### Issue Description

I am unable to run any test for Scan right now.

If it doesn't timeout the build output states that the `Tests Fail` which I think is Xcode's cryptic way of saying it took long to build.

The simulator is launched but it is black and never loads. XCTool and Xcode are able to run unit tests but xcodebuild is not.

### Configuration Files

This was ran manually using `scan`.

### Environment

fastlane version (fastlane -v): fastlane 1.66.0
Do you use bundler, rbenv or rvm? RVM

**Comment by TKBurner** [comment]
Hi @jcampbell05! Great to hear from you again. Can you reformat your post to this template? https://github.com/fastlane/fastlane/blob/master/ISSUE_TEMPLATE.md 

Also, what version of xCode are you using? Did this work before the update to xCode 7.3? Thanks!

**Comment by kcharwood** [comment]
I'm seeing this issue too. I'm telling scan to use an iPhone 6, and a 6s Plus is launching on one of my ci machines.

```
[11:12:04]: ▸ 2016-03-24 11:12:04.146 xcodebuild[1962:11392] [MT] IDETestOperationsObserverDebug: (207489D3-8F21-49DC-866E-16FB5FC9E848) Beginning test session 207489D3-8F21-49DC-866E-16FB5FC9E848 with Xcode 7D175 on target <DVTiPhoneSimulator: 0x7ff9bdf39b80> {
[11:12:04]: ▸ SimDevice: SimDevice : iPhone 6 (82D8FCE6-DF47-44A0-AC64-DB9507E1D41E) : state={ Shutdown } deviceType={ SimDeviceType : com.apple.CoreSimulator.SimDeviceType.iPhone-6 } runtime={ SimRuntime : 9.3 (13E230) - com.apple.CoreSimulator.SimRuntime.iOS-9-3 }
[11:12:04]: ▸ } (9.3 (13E230))
[11:12:04]: ▸ 2016-03-24 11:12:04.149 xcodebuild[1962:11401]  IDETestOperationsObserverDebug: Writing diagnostic log for test session to:
[11:12:04]: ▸ /Users/machine-name/Library/Developer/Xcode/DerivedData/MMSDK-apvoxwjemajcwocjwqiyfblgwalr/Logs/Test/C6674BD8-6016-4D0A-84EE-84B9BFA60EED/Session-2016-03-24_11:12:04-B9k7sp.log
```

**Comment by jcampbell05** [comment]
@TKBurner Sure I'll reformat.

It didn't work in Xcode 7.2 either.

**Comment by TKBurner** [comment]
Thanks for following up @jcampbell05. Can you send me there error you get when you run xcodebuild?

Thanks!

**Comment by Gladkov-Art** [comment]
I have the same issue. The last line in my console is:
 `xcodebuild[4152:18397]  iPhoneSimulator: Timed out waiting 120 seconds for simulator to boot, current state is 1.`

**Comment by TKBurner** [comment]
@Gladkov-Art Thanks for letting me know. Does the simulator launch and open your app during this time span?

**Comment by Gladkov-Art** [comment]
@TKBurner no, it doesn't. Scan opens default simulator with black screen.

**Comment by TKBurner** [comment]
@Gladkov-Art Thanks for following up. What happens if you clear the data from your simulator?

Thanks!

**Comment by kcharwood** [comment]
To be clear, when I saw this issue, I was clearly passing in an iPhone 6 9.3 simulator, scan was sending the correct UUID for that simulator, but an iPhone 6s Plus simulator was launched with a black screen.

I've since nuked that machine and started over with a new xcode install.

**Comment by Gladkov-Art** [comment]
@TKBurner, Thanks. I tried, It didn't help.

**Comment by Gladkov-Art** [comment]
@kcharwood thanks for your expirience. Probably you have tried everything before it. I have reinstalled OS X recently, so is there anything else I can try?

**Comment by kcharwood** [comment]
Ok I made it back to my build machine that was failing last week, and its still failing this morning.

Here is all the logs I have:

[Logs.zip](https://github.com/fastlane/fastlane/files/191817/Logs.zip)

* *fastlane_console.log*: The terminal output for running fastlane
* *Simulator_Diagnostic_Log.log*: The diagnostic log referenced in the fastlane output
* *Xcodebuild.log*: The raw log generated by xcode build.

If I wait long enough, I get 

> iPhoneSimulator: Timed out waiting 120 seconds for simulator to boot, current state is 1.

Note I took the raw Xcode build command, and ran it directly from terminal, and it also fails, so I'm not sure this is even a fastlane issue but an Xcode issue?

**Comment by kcharwood** [comment]
I just ran `snapshot reset_simulators`, and I can clearly see that _wrong_ simulator is being launched by Xcode.

**Comment by kcharwood** [comment]
More data.

If I run the command generated by `scan`

```
xcodebuild -scheme 'MMEcommSDK' -project 'MMEcommSDK/UAEcommSDK.xcodeproj' -destination 'platform=iOS Simulator,id=AB1B7477-BA8B-45C0-90EE-CD889E0D49A0' build test 
```

It fails to build with the timeout error. If I remove `build` from the command, and only leave `test`, it succeeds

```
xcodebuild -scheme 'MMEcommSDK' -project 'MMEcommSDK/MMEcommSDK.xcodeproj' -destination 'platform=iOS Simulator,id=AB1B7477-BA8B-45C0-90EE-CD889E0D49A0' test 
```

**Comment by TKBurner** [comment]
duplicate of #1926. Closing this thread and moving over there.

**Comment by kcharwood** [comment]
I don't think this is related to #1926

**Comment by Gladkov-Art** [comment]
Agree with @kcharwood

**Comment by kcharwood** [comment]
Note at this time, I can think I can work around this "issue" by using the `skip_build` parameter on `scan`. I still don't know that the actual issue is though.

**Comment by Gladkov-Art** [comment]
@kcharwood, how do you build project before testing it?

**Comment by TKBurner** [comment]
@kcharwood How long does your app take to build normally?

@Gladkov-Art Are you getting the same exact error message?

Thanks for working with me!

**Comment by Gladkov-Art** [comment]
@TKBurner yes, exactly the same.

**Comment by jcampbell05** [comment]
Its worth mentioning I was able to get tests to run using XCTool or running them manually from Xcode, so they are doing something magical which causes this to work.

**Comment by kcharwood** [comment]
I filed radar [25412159](http://openradar.appspot.com/25412159)

**Comment by TKBurner** [comment]
@kcharwood Thanks for filling a radar. Let us know if you get any updates.

**Comment by jshier** [comment]
This is an issue on TravisCI as well. I suspect the underlying issue is that the simulator can sometimes fail to launch correctly, especially after they've just been created. Usually quitting the simulator and then manually launching it gets it to work. Unfortunately this is obviously impossible in a CI situation you don't control.

**Comment by TKBurner** [comment]
@jshier Thanks for following up. We are going to look into this over here. We'll update on this thread 👍

**Comment by jshier** [comment]
@TKBurner I've been encountering the `scan` hang issue on our local Jenkins bots. I know the simulator works on those machines as I can run them manually. There is no simulator timeout, just this:
```
[14:16:01]: Running Tests: ▸ Touching Tests.xctest
Build timed out (after 10 minutes). Marking the build as aborted.
Build was aborted
```
Retrying the job usually gets it to work.

**Comment by TKBurner** [comment]
@jshier Thanks for bumping this. I'll flag this to the team 👍

**Comment by kalle** [comment]
I have similar problems with `snapshot` and I think fastlane/snapshot has something do to with it and not only xcode.


1. When running `env SNAPSHOT_DERIVED_DATA_PATH=~/tmp/snapshots_ostl snapshot` an empty (black only) simulator is opening of the same kind that the last open one (if I had iPad2 open but want to run run tests on iPhone6, the iPad2 one opens) and I get the "120 seconds timeout" error. Rerunning the `xcodebuild ... build test` command manually yields the same result.
1. I kill the Simulator and the `/Applications/Xcode.app/Contents/Developer/Library/PrivateFrameworks/CoreSimulator.framework/Versions/A/XPCServices/com.apple.CoreSimulator.CoreSimulatorService.xpc/Contents/MacOS/com.apple.CoreSimulator.CoreSimulatorService` process.
1. Now if I manually open the simulator I want with `open -a Simulator --args -CurrentDeviceUDID 1CCE0710-AF33-44D5-B55A-6CF77823DEE2` it starts up and boots into the home screen.
1. Now I can run the `xcodebuild ... build test` command from snapshot successfully.
1. But if I rerun the `snapshot` command it will kill the simulator and restart a new one that doesn't boot into the home screen and instead shows a black page again.

This makes me think the problem is indeed in fastlane/snapshot or in the simulator restart and that `xcodebuild .. build test` has nothing to do with it.

**Comment by TKBurner** [comment]
@kalle Thanks for sharing those additional details. I'm going to bump this thread internally. 👍

**Comment by mfurtak** [comment]
Hi folks! 👋 

I'm working on investigating this right now. Here's my system setup:

```
Mac OS X: 10.11.4
xcodebuild: Xcode 7.3, Build version 7D175
snapshot: 1.12.1
scan 0.6.0
```

Just starting with the most recent info (which was about snapshot, not scan):

When running `env SNAPSHOT_DERIVED_DATA_PATH=~/tmp/snapshots_ostl snapshot` against one of my local projects, I do not have any problems 😞

My steps were:

1. Run the app on an iPad 2 simulator through Xcode
1. Quit the simulator
1. Run `env SNAPSHOT_DERIVED_DATA_PATH=~/tmp/snapshots_ostl snapshot` against my project
    * The first simulator to boot and run tests is the iPhone 4s
    * At no point does an iPad 2 simulator start up, and none of the simulators fail to boot
1. Five devices worth of screenshots complete successfully

So... therein lies our challenge - we need to be able to reproduce this in order to fix it :smile:

What would be really _really_ helpful would be if someone was able to share a small test project that exhibits this behavior with either scan or snapshot that we can pull down and use.

If the broken behavior can't be triggered by a particular project setup, and it's somehow related to the state of the various simulators on a development machine (as it sounds to me like it is), without some clues about how to get our local simulators into that bad state, we're going to continue to struggle to reproduce this.

I think @kcharwood's [Apple radar](http://openradar.appspot.com/25412159) is important here, because it shows that the problem can be triggered without having `scan` or `snapshot` in the mix.

If folks are interested in seeing whether omitting `build` from the generated `scan` command can help work around this problem (as described in the radar) that might be something I could cook up.

What do folks think?

**Comment by kalle** [comment]
Hello @mfurtak 

Thanks for looking into this!

I have created a simple project that have the same problems as my real project and the workaround with `open -a Simulator ...` etc works. You can find it at https://github.com/kalle/FLbug. 

Perhaps this is locale based? I use Swedish locale which uses `,` (comma) instead of `.` (dot) as decimal separator. I tried to set `LANG`, `LC_*` and `__CF_USER_TEXT_ENCODING` to `en` and `US` where applicable and run `snapshot` again but still got the same result. I'm not sure how ruby handle locale related things or if it is even the source of the problem here. But since `snapshot` modifies configuration files it might be worth looking into.

Anyway, since I really needed the screenshots I wrote a hacky clone of `snapshot` in python. I used `open -a Simulator ...` and liberal use of `killall Simulator`. Perhaps you can use that to instead of `simctl` if the root cause seems to be on Apple's side?

**Comment by mfurtak** [comment]
@kalle Fantastic! Thank you! I will take a look at it.

**Comment by mfurtak** [comment]
This is excellent! It reproduces every time with this project. Thank you so much! 🎉 

I will post updates about my investigation here.

**Comment by kalle** [comment]
@mfurtak That is great news! I was almost certain that it was something outside the project itself.

Also, I just updated to `Xcode 7.3.1, Build version 7D1014`, but still got the same problem.

**Comment by mfurtak** [comment]
OK, I have found that if I introduce an `open -a Simulator ...` call to open the correct simulator before executing the `xcodebuild` command, that the simulators will not fail to boot, and the tests will pass.

This is not a wholly satisfying solution because it doesn't really explain what is causing the combination of `xcodebuild` and the simulators to get into a bad state without this.

That being said, if I put up a PR with this change, would folks be interested in trying it to see if it makes things work?

**Comment by kalle** [comment]
Sure, I can test it if you provide me with detailed instructions on how to build etc (not a ruby guy).

Yeah, this is more of a workaround than a proper solution.

I still find it odd that this is project specific though. If you recreate my project from scratch on your machine would you still get the problem? It is just a "single page app" project, the only "weird" thing I did was to make the test project a module to get the swift bridge to work. I also recreated it in swift (in case it was the module thing that triggered it) but got the same problem.

**Comment by mfurtak** [comment]
OK, getting a Ruby dev environment together is not always easy, so I won't ask you to do that just yet. :smile:

I find it odd as well, but yes. I just created a brand new Objective-C Single View project, brought in the SnapshotHelper and Swift bridge, and set up some simple snapshots. My project does not have any problems, but I can switch to yours and reproduce it no problem 😞

I'll definitely be trying to spend some more time figuring out the differences between them.

**Comment by kalle** [comment]
I made a new repo for you: https://github.com/kalle/FLBug2

It is a swift single page project with UI testing that I didn't modify in any way other than adding a `Snapfile`. It will probably make it easier for you to figure out the difference between my project and yours.

**Comment by mfurtak** [comment]
I'm still focused on `snapshot` at the moment, but here's what I've found:

The offending behavior that triggers the `xcodebuild` + Simulator lockup is the `killall` we apply to the Simulators. We kill all open simulators so that our zoom and keyboard fixes (which are `defaults write`s) get picked up by the next Simulator launch. I've confirmed that if I remove the `killall` there is no hang and the tests pass.

However, if we need the tests to run on the same Simulator that was already open (and had a different zoom level, for example) then the fixes are not properly applied without the `killall`.

**Comment by kalle** [comment]
OK, I see.

After you have done `killall` do you wait until the killed simulator has reached state `Shutdown` (as reported by `simctl list`) before starting again? `killall` sends a `SIGTERM` by default and the simulator is closing down in an orderly fashion for a few seconds (is in state `Shutting down` or similar).

**Comment by mfurtak** [comment]
@kalle I've released [`snapshot` 1.12.2](https://github.com/fastlane/fastlane/releases/tag/snapshot%2F1.12.2) that includes the previously discussed workaround behind a feature switch.

Could you please upgrade your `snapshot` with:

```bash
gem update snapshot
```

and then try running:

```bash
FASTLANE_EXPLICIT_OPEN_SIMULATOR=1 snapshot
```

and let me know if allows your test runs to succeed? Thanks! 🚀

**Comment by kalle** [comment]
@mfurtak Yes! Your change did work! Thank you so much! 👍

**Comment by alexanderjrobinson** [comment]
After much research, my issue with the simulator time out was due to using launchd to launch the build agent for my CI server. As of Xcode 6 processes started from launchd can't connect to the simulator because they don't have a GUI context. If anyone is having the same issue here are some related posts that I used to fix my issue. 

http://www.openradar.appspot.com/18282084
https://gist.github.com/rbuussyghin/0d7473a2692bdf6e0ef7

**Comment by TKBurner** [comment]
@alexanderjrobinson Thanks for sharing! 

Based on the release above I'm going to mark this awaiting reply. If we don't get any new reports we'll close this out 👍

**Comment by mfurtak** [comment]
@jcampbell05 I've released [`scan` 0.6.1](https://github.com/fastlane/fastlane/releases/tag/scan%2F0.6.1) that includes this speculative workaround behind a feature switch.

Could you please upgrade your `scan` with:

```bash
gem update scan
```

and then try running:

```bash
FASTLANE_EXPLICIT_OPEN_SIMULATOR=1 scan
```

and let me know if allows your test runs to succeed? Thanks! 🚀

**Comment by jcampbell05** [comment]
@TKBurner this report this applies to `scan`, please could you wait until we confirm the fix for scan works then we can close :)

**Comment by jcampbell05** [comment]
@mfurtak  @TKBurner 
I can confirm this works when:

- Ran with no simulator running and scan was able to launch it.
- Ran with a simulator previously booted and scan was able to use it.
- Ran with a simulator previously booted but a different device and scan was able to switch to a different simulator.

Closing since this seems to be fixed, thanks for all your hard work everyone, @alexsup and I owe you a beer or a club mate ;) !

**Comment by KrauseFx** [comment]
@jcampbell05 Club Mate, haha 👍

**Comment by mr-brandt** [comment]
Hello,
last weekend i ran into the issue that snapshot states "iPhoneSimulator: Timed out waiting 120 seconds for simulator to boot, current state is 1.". I tried the workaround by setting the environment variable beforehand as stated above by calling `FASTLANE_EXPLICIT_OPEN_SIMULATOR=1 snapshot`.

I tried the following things:

- reinstall simulators
- reinstall app
- checked that i am using the latest version of snapshot

What i still see is that the output says
`Explicitly opening simulator for device: iPhone 4s`
but the iPhone 6s Plus simulator is launched and then the last message states
`2016-06-13 09:55:19.672 xcodebuild[25888:434446]  iPhoneSimulator: Timed out waiting 120 seconds for simulator to boot, current state is 1.`

Is there still an issue with the current release of snapshot or am i doing something wrong?

Thanks!

**Comment by TKBurner** [comment]
@mr-brandt Thanks for reaching out. Can you submit a new issue for this?

Also are you using the erase_simulator: true and reinstall_app: true parameters.

Finally share the full output on the new issue you create. Thanks!

**Comment by rafaelnobrepd** [comment]
Hi @mfurtak et all, I've been having sporadic issues very similar to this when running scan on travis, is the FASTLANE_EXPLICIT_OPEN_SIMULATOR=1 workaround still relevant ? Or should this have been fixed by now ? I'm using scan 0.10.1. I can't reproduce this locally at all, but travis regularly gets stuck on ```▸ Touching MedicamentosTests.xctest
** TEST FAILED **
[17:51:04]: Exit status: 65```

**Comment by mfurtak** [comment]
@rafaelnobrepd The behavior triggered by `FASTLANE_EXPLICIT_OPEN_SIMULATOR=1` is still not enabled by default, so if you're not using that environment variable feature switch, it is definitely worth trying 👍

**Comment by rafaelnobrepd** [comment]
Thanks, it seems that fixed the issue!

**Comment by Sea-Eagle** [comment]
@mr-brandt  have resolved the issue I'm facing same issue

**Comment by TKBurner** [comment]
@Sea-Eagle Did you try @mfurtak's suggestion? Thanks!

**Comment by Sea-Eagle** [comment]
Yes I have tried FASTLANE_EXPLICIT_OPEN_SIMULATOR=1  but for this I'm getting error "Wrong number of arguments (given 1 expected 2)"  even if i try with 2 I'm getting same error

**Comment by TKBurner** [comment]
@Sea-Eagle Can you share the full input? It should be like `ENV_NAME="123" fastlane beta`

**Comment by Sea-Eagle** [comment]
ENV_NAME="FASTLANE_EXPLICIT_OPEN_SIMULATOR=1" fastlane snapshot
ENV_NAME="FASTLANE_EXPLICIT_OPEN_SIMULATOR=1" fastlane beta

both i have tried.

the output is 


The following build commands failed:
Check dependencies
(1 failure)
[08:11:56]: Exit status: 65
[08:11:56]: Tests failed - check out the log above
[08:11:56]: Tests failed, re-trying 1 out of 2 times
[08:11:56]: Patching '/Users/Prajwalkumar/Library/Preferences/com.apple.iphonesimulator.plist' to scale simulator to 100%
[08:11:57]: -----------------------------------
[08:11:57]: --- iPad Pro (9.7-inch) - it-IT ---
[08:11:57]: -----------------------------------
[08:11:57]: $ set -o pipefail && xcodebuild -scheme SnapshotLoginApplication -project ./SnapshotLoginApplication.xcodeproj -derivedDataPath '/var/folders/x1/ndgsd09j74xcrpcrcj6bjqvw0000gp/T/snapshot_derived20160915-12210-bs13r3' -destination 'platform=iOS Simulator,id=24A3B42D-A484-404E-8385-11E738D10785,OS=10.0' FASTLANE_SNAPSHOT=YES build test | tee /Users/Prajwalkumar/Library/Logs/snapshot/SnapshotLoginApplication-SnapshotLoginApplication.log | xcpretty 
[08:11:57]: ▸ Loading...
[08:11:58]: ▸ Building SnapshotLoginApplication/SnapshotLoginApplication [Debug]
[08:11:58]: ▸ Check Dependencies
[08:11:58]: ▸ Build Succeeded
[08:11:58]: ▸ 2016-09-15 08:11:58.646 xcodebuild[14840:212188] [MT] IDETestOperationsObserverDebug: (022B3F1B-5CE0-4B14-98CA-833FD0CCAAEC) Beginning test session SnapshotLoginApplicationUITests-022B3F1B-5CE0-4B14-98CA-833FD0CCAAEC at 2016-09-15 08:11:58.645 with Xcode 8A218a on target <DVTiPhoneSimulator: 0x7fd9aceb5180> {
[08:11:58]: ▸ SimDevice: SimDevice : iPad Pro (9.7-inch) (24A3B42D-A484-404E-8385-11E738D10785) : state={ Shutdown } deviceType={ SimDeviceType : com.apple.CoreSimulator.SimDeviceType.iPad-Pro--9-7-inch- } runtime={ SimRuntime : 10.0 (14A345) - com.apple.CoreSimulator.SimRuntime.iOS-10-0 }
[08:11:58]: ▸ } (10.0 (14A345))
[08:11:58]: ▸ 2016-09-15 08:11:58.646 xcodebuild[14840:212197]  IDETestOperationsObserverDebug: Writing diagnostic log for test session to:
[08:11:58]: ▸ /var/folders/x1/ndgsd09j74xcrpcrcj6bjqvw0000gp/T/snapshot_derived20160915-12210-bs13r3/Logs/Test/DE37FD34-B9A8-4535-94AB-98896C99291E/Session-SnapshotLoginApplicationUITests-2016-09-15_081158-8gGMuJ.log
[08:11:58]: ▸ Building SnapshotLoginApplication/SnapshotLoginApplication [Debug]
[08:11:58]: ▸ Check Dependencies
[08:11:58]: ▸ Building SnapshotLoginApplication/SnapshotLoginApplicationUITests [Debug]
[08:11:58]: ▸ Check Dependencies
[08:11:58]: ▸ 2016-09-15 08:11:58.822 xcodebuild[14840:212188] Error Domain=IDETestOperationsObserverErrorDomain Code=3 "Test operation was canceled. If you believe this error represents a bug, please attach the log file at /var/folders/x1/ndgsd09j74xcrpcrcj6bjqvw0000gp/T/snapshot_derived20160915-12210-bs13r3/Logs/Test/DE37FD34-B9A8-4535-94AB-98896C99291E/Session-SnapshotLoginApplicationUITests-2016-09-15_081158-8gGMuJ.log" UserInfo={NSLocalizedDescription=Test operation was canceled. If you believe this error represents a bug, please attach the log file at /var/folders/x1/ndgsd09j74xcrpcrcj6bjqvw0000gp/T/snapshot_derived20160915-12210-bs13r3/Logs/Test/DE37FD34-B9A8-4535-94AB-98896C99291E/Session-SnapshotLoginApplicationUITests-2016-09-15_081158-8gGMuJ.log}
[08:11:58]: ▸ Testing failed:
[08:11:58]: ▸ “Use Legacy Swift Language Version” (SWIFT_VERSION) is required to be configured correctly for targets which use Swift. Use the [Edit > Convert > To Current Swift Syntax…] menu to choose a Swift version or use the Build Settings editor to configure the build setting directly.
[08:11:58]: ▸ ** TEST FAILED **
[08:11:58]: ▸ The following build commands failed:
[08:11:58]: ▸ Check dependencies
[08:11:58]: ▸ (1 failure)
▸ Building SnapshotLoginApplication/SnapshotLoginApplication [Debug]
▸ Check Dependencies
▸ Build Succeeded
2016-09-15 08:11:58.646 xcodebuild[14840:212188] [MT] IDETestOperationsObserverDebug: (022B3F1B-5CE0-4B14-98CA-833FD0CCAAEC) Beginning test session SnapshotLoginApplicationUITests-022B3F1B-5CE0-4B14-98CA-833FD0CCAAEC at 2016-09-15 08:11:58.645 with Xcode 8A218a on target <DVTiPhoneSimulator: 0x7fd9aceb5180> {
SimDevice: SimDevice : iPad Pro (9.7-inch) (24A3B42D-A484-404E-8385-11E738D10785) : state={ Shutdown } deviceType={ SimDeviceType : com.apple.CoreSimulator.SimDeviceType.iPad-Pro--9-7-inch- } runtime={ SimRuntime : 10.0 (14A345) - com.apple.CoreSimulator.SimRuntime.iOS-10-0 }
} (10.0 (14A345))
2016-09-15 08:11:58.646 xcodebuild[14840:212197]  IDETestOperationsObserverDebug: Writing diagnostic log for test session to:
/var/folders/x1/ndgsd09j74xcrpcrcj6bjqvw0000gp/T/snapshot_derived20160915-12210-bs13r3/Logs/Test/DE37FD34-B9A8-4535-94AB-98896C99291E/Session-SnapshotLoginApplicationUITests-2016-09-15_081158-8gGMuJ.log
▸ Building SnapshotLoginApplication/SnapshotLoginApplication [Debug]
▸ Check Dependencies
▸ Building SnapshotLoginApplication/SnapshotLoginApplicationUITests [Debug]
▸ Check Dependencies
2016-09-15 08:11:58.822 xcodebuild[14840:212188] Error Domain=IDETestOperationsObserverErrorDomain Code=3 "Test operation was canceled. If you believe this error represents a bug, please attach the log file at /var/folders/x1/ndgsd09j74xcrpcrcj6bjqvw0000gp/T/snapshot_derived20160915-12210-bs13r3/Logs/Test/DE37FD34-B9A8-4535-94AB-98896C99291E/Session-SnapshotLoginApplicationUITests-2016-09-15_081158-8gGMuJ.log" UserInfo={NSLocalizedDescription=Test operation was canceled. If you believe this error represents a bug, please attach the log file at /var/folders/x1/ndgsd09j74xcrpcrcj6bjqvw0000gp/T/snapshot_derived20160915-12210-bs13r3/Logs/Test/DE37FD34-B9A8-4535-94AB-98896C99291E/Session-SnapshotLoginApplicationUITests-2016-09-15_081158-8gGMuJ.log}

Testing failed:
“Use Legacy Swift Language Version” (SWIFT_VERSION) is required to be configured correctly for targets which use Swift. Use the [Edit > Convert > To Current Swift Syntax…] menu to choose a Swift version or use the Build Settings editor to configure the build setting directly.
** TEST FAILED **

**Comment by Sea-Eagle** [comment]
@TKBurner Thanks its working fine now. I just updated my project build settings

**Comment by TKBurner** [comment]
@Sea-Eagle Awesome! Really glad you got up and running. 👍
